src/main/resources/application.conf

```diff
@@ -36,6 +36,11 @@ workspace {
   workspacesPath="/workspaces"
 }
 
+entities {
+  baseUrl="https://rawls-dev.broadinstitute.org"
+  entitiesPath="/entities"
```

Comment:
This should be in the workspace configuration block since it is pointing to rawls.